toolkit/tools/internal/pkggraph/pkggraph.go

```diff
@@ -326,7 +333,21 @@ func (g *PkgGraph) addToLookup(pkgNode *PkgNode, deferSort bool) (err error) {
 			return
 		}
 		existingLookup = &LookupNode{nil, nil}
+		logger.Log.Debugf("Before adding %s to lookup table", pkgName)
+		//check if pkgName exists in lookup table
+		if _, ok := g.lookupTable()[pkgName]; ok {
+			logger.Log.Debugf("Key exists for %s", pkgName)
+		} else {
+			logger.Log.Debugf("Key doesn't exist for %s", pkgName)
+		}
 		g.lookupTable()[pkgName] = append(g.lookupTable()[pkgName], existingLookup)
+		logger.Log.Debugf("Added %s to lookup table", pkgName)
+		//check if pkgName exists in lookup table
+		if _, okk := g.lookupTable()[pkgName]; okk {
```

Comment:
```suggestion
		if _, ok = g.lookupTable()[pkgName]; ok {
```
I think this reads better, the `okk` looks odd to me.